modules/fb_instant_articles_display/fb_instant_articles_display.info

```diff
@@ -2,6 +2,6 @@ name = Facebook Instant Articles Display
 description = Display module for Facebook Instant Articles.
 package = Facebook Instant Articles
 core = 7.x
-configure = admin/config/services/fb-instant-articles/display
 dependencies[] = ctools
 dependencies[] = fb_instant_articles
+scripts[] = js/admin.js
```

Comment:
Yeah, this was a merge blunder (I had removed it here, but it came back during a rebase conflict resolution). I'll remove it.